app/src/lib/parser/index.js

```diff
@@ -1,10 +1,8 @@
 import {pipe} from '$lib/pipeline'
 import {tokenize_input} from './tokenize'
-import {check_for_pronouns} from './pronoun_rules'
 import {perform_form_lookups, perform_ontology_lookups} from '$lib/lookups'
-import {check_pairings} from './pairings'
 import {clausify, flatten_sentences} from './clausify'
-import {CHECKER_RULES, LOOKUP_RULES, PART_OF_SPEECH_RULES, SYNTAX_RULES, TRANSFORM_RULES, rules_applier} from '$lib/rules'
+import {CHECKER_RULES, LOOKUP_RULES, PART_OF_SPEECH_RULES, SYNTAX_RULES, TRANSFORM_RULES, PRONOUN_RULES, rules_applier} from '$lib/rules'
```

Comment:
if this is starting to feel unwieldy, you could refactor it into a `Rules` object where the properties are the different types.